src/datajudge/constraints/base.py

```diff
@@ -121,12 +123,12 @@ def __init__(
         self,
         ref: DataReference,
         *,
-        ref2: Optional[DataReference] = None,
-        ref_value: Optional[Any] = None,
-        name: Optional[str] = None,
-        output_processors: Optional[
-            Union[OutputProcessor, List[OutputProcessor]]
-        ] = output_processor_limit,
+        ref2: DataReference | None = None,
+        ref_value: Any = None,
```

Comment:
Not critical, but the change from optional any to any feels wrong. But probably is an issue that Any is too vague to start with.

Reply:
I see your point. Do you suggestion yet?

Reply:
I have no suggestion. Well, one, but it's quite vague. To create a type that reflects what a `ref_value` can be. But it feels too much work for little gain.